bin/run_ci_dataset.sh

```diff
@@ -35,17 +35,19 @@ print_error() {
 
 usage() {
     print_error
-    print_error "Usage: $0 -d DATASET [-h]"
+    print_error "Usage: $0 -d DATASET [-g NUM] [-h]"
     print_error
     print_error "Specifc options:"
```

Comment:
(Just noticed this going by in my mail and though “hey, maybe this'd be a good opportunity to fix that typo...”)